docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -176,6 +176,36 @@ export default function App() {
 
 In the terminal, run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the React Query plugin. This will open the plugin's web interface, displaying queries as they are used in your app.
 
+### Redux DevTools
+
+Based on the full [Redux DevTools](https://github.com/reduxjs/redux-devtools/) (from the Chrome extension) but as an Expo Dev Plugin.
+
+- Easy to install and run.
+- Live list of actions and how it affects the state.
+- Ability to rewind, replay, and dispatch actions from the devtools.
+
+To use the plugin, start by installing the package:
+
+<Terminal cmd={['$ npx expo install redux-devtools-expo-dev-plugin']} />
+
+If you're using `@reduxjs/toolkit` then you need to modify your call to `configureStore` to disable the built in devtools (by passing in `devTools: false`) and add in the Expo Devtools plugin enhancer (by concatenating the `devToolsEnhancer()`).
+
+Your call to `configureStore` will end up looking like this:
+
+```javascript store.js
+import devToolsEnhancer from 'redux-devtools-expo-dev-plugin';
+
+const store = configureStore({
+  reducer: rootReducer,
+  devTools: false,
+  enhancers: getDefaultEnhancers => getDefaultEnhancers().concat(devToolsEnhancer()),
+});
+```
+
+In the terminal, run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the Tinybase plugin. This will open the plugin's web interface, displaying the actions and contents of your store as actions are dispatched.
+
+For full installation instructions, including if your using `redux` directly rather than `@reduxjs/toolkit`, [see the project README](https://github.com/matt-oakes/redux-devtools-expo-dev-plugin).
```

Comment:
```suggestion
For complete installation and usage instructions, including if you're using `redux` directly rather than `@reduxjs/toolkit`, [see the project's README](https://github.com/matt-oakes/redux-devtools-expo-dev-plugin).
```

Since the readme of the plugin also include more than installation instructions, let's highlight that here. Also, let's fix typos.